build/webpack/common.js

```diff
@@ -38,3 +38,78 @@ function getListOfExistingModulesInOutDir() {
     return files.map((filePath) => `./${filePath.slice(0, -3)}`);
 }
 exports.getListOfExistingModulesInOutDir = getListOfExistingModulesInOutDir;
+
+const bundleConfiguration = {
+    // We are bundling for both Web and Desktop.
+    webAndDesktop: 'webAndDesktop',
+    // We are bundling for both Web only.
+    web: 'web',
+    // We are bundling for both Desktop only.
+    desktop: 'desktop'
+};
+/**
+ * Gets the bundle configuration based on the environment variable.
+ * @return {'webAndDesktop' | 'web' | 'desktop'}
+ */
+function getBundleConfiguration() {
+    if (process.env.VSC_VSCE_TARGET === 'web') {
+        return bundleConfiguration.web;
+    } else if (process.env.VSC_VSCE_TARGET === undefined) {
+        // Building locally or on Github actions, when we're not creating platform specific bundles.
+        return bundleConfiguration.webAndDesktop;
+    } else {
+        return bundleConfiguration.desktop;
+    }
+}
+
+function getZeroMQPreBuildsFoldersToKeep() {
+    // Possible values of 'VSC_VSCE_TARGET' include platforms supported by `vsce package --target`
+    // See here https://code.visualstudio.com/api/working-with-extensions/publishing-extension#platformspecific-extensions
+    const vsceTarget = process.env.VSC_VSCE_TARGET;
```

Comment:
Previously we'd deleted files from the `node_modules` folder, now we copy the required files into the output directory